Doppler.Sap.Test/BillingRequestHandlerTest.cs

```diff
@@ -108,20 +98,10 @@ public async Task BillingRequestHandler_ShouldBeCreateBillingInSap_WhenQueueHasO
                 sapConfigMock.Object,
                 Mock.Of<ILogger<BillingRequestHandler>>(),
                 sapServiceSettingsFactoryMock.Object,
-                httpClientFactoryMock.Object,
+                httpClientFactoryMock,
```

Comment:
I am not sure if it is well done or not, but in general, we tend to use the suffix `Mock` for mock created wrappers (the old class), and nothing for the doubles:

```suggestion
                httpClientFactory,
```